robot/packages/cloud_package_manager.go

```diff
@@ -276,7 +276,7 @@ func (m *cloudManager) Cleanup(ctx context.Context) error {
 			}
 			_, expectedToExist := expectedPackageDirectories[packageDirName]
 			if !expectedToExist {
-				m.logger.Debug("Removing old package", packageDirName)
+				m.logger.Debugf("Removing old package %s", packageDirName)
```

Comment:
not totally sure what this should be, since I'm not sure what form `packageDirName` is in

Reply:
This is right.